openmc/deplete/msr.py

```diff
@@ -0,0 +1,183 @@
+from abc import ABC, abstractmethod
+from collections import OrderedDict
+from copy import deepcopy
+from warnings import warn
+from numbers import Real
+
+import numpy as np
+import h5py
+
+from openmc.checkvalue import check_type, check_value, check_less_than, \
+check_iterable_type, check_length
+from openmc import Materials, Material, Cell
+from openmc.search import _SCALAR_BRACKETED_METHODS, search_for_keff
+from openmc.data import atomic_mass, AVOGADRO, ELEMENT_SYMBOL
+import openmc.lib
+
+class MsrContinuous:
```

Comment:
Probably should add a reference to a paper which presents the ideas that this class implements, likely Aufiero's paper. https://www.sciencedirect.com/science/article/pii/S0022311513008507

Reply:
Good idea